docs/resources/event.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: "wavefront"
+page_title: "Wavefront: Event"
+description: |-
+Provides a Wavefront Event resource.  This allows events to be created, updated, and deleted.
+---
+
+# Resource : wavefront_event
+
+Provides a Wavefront Event resource.  This allows events to be created, updated, and deleted.
```

Comment:
Provides a Wavefront event resource. 

+ remove the extra space between sentences